tests/main/mount-ns/google.ubuntu-18.04-64/HOST.expected.txt

```diff
@@ -10,14 +10,15 @@
 +0:+1 / /proc/sys/fs/binfmt_misc rw,relatime shared:+1 - autofs systemd-1 rw,fd=0,pgrp=1,timeout=0,minproto=5,maxproto=5,direct,pipe_ino=0
 +0:+1 / /run rw,nosuid,noexec,relatime shared:+1 - tmpfs tmpfs rw,size=VARIABLE,mode=755
 +0:+1 / /run/lock rw,nosuid,nodev,noexec,relatime shared:+1 - tmpfs tmpfs rw,size=VARIABLE
++0:+1 / /run/qemu rw,nosuid,nodev,relatime shared:+1 - tmpfs none rw,mode=755
```

Comment:
ideally we would make these changes separate from this PR, since they are large and not actually related to the other changes here and may distract/confuse about the changes in this PR

Reply:
But then we'd break the master branch, unless I'm missing something. Or is this test currently failing in master too?

Reply:
yes, the mount-ns:inherit (or maybe the reboot variant) test is currently failing on master a lot